src/oic/utils/sdb.py

```diff
@@ -613,6 +474,7 @@ def __init__(
 
         self.access_token = self.token_factory["access_token"]
         self.token = self.access_token
+        self.sm_salt = sm_salt or rndstr(32)
```

Comment:
This probably should be set first and then set the `sm_salt` on `_db`?

Reply:
Ah, yes!